Tryouts/Prototypes/Shell/MainWebWindowOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Shell
+{
+    public sealed class MainWebWindowOptions
+    {
+        public double? Height { get; set; }
+        public string? Title { get; set; }
+        public string? Url { get; set; }
+        public double? Width { get; set; }
+
+        public double DefaultHeight { get; set; } = 450d;
```

Comment:
These should be static constants with non-nullable type (`public const double DefaultHeight = 450` etc.)